bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +268,25 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
+        if valid_email_check:
+            result_message = 'Body contains these invalid emails:\n'
+            invalid_emails = False
+            for line in body:
+                for email in re.findall(r'\S*@\S*', line):
```

Comment:
This regex will also find things that could not be intended to be E-Mail adresses, like for example Github user names, e.g. `@NiklasMM`

I would require at least one charater on each side and maybe even a dot: `r'\S+@\S+\.\S+`